### ISSUE TITLE
 imageChooser

### ISSUE THREAD

**Issue by wsry888** [yolain/ComfyUI-Easy-Use#169]
When using the easy imageChooser node, the entire workflow becomes stuck and cannot proceed, necessitating the interruption of the current workflow.

**Comment by yolain** [comment]
Is the page stuck or is the queue paused？

**Comment by wsry888** [comment]
The entire page cannot be moved, and the workflow is stuck. The way to resolve that is to cancel the current queue.

**Comment by PrometheusDante** [comment]
Can confirm the same behaviour. Panning no longer works, canceling the queue does cancel properly, but the page remains stuck until refreshed. The cmd-window does not show any errors either, it just gets stuck. Went back to the larger preview chooser to bypass this problem for now.

**Comment by wsry888** [comment]
Yes, the page needs to be refreshed, but there are no error messages in the console.

**Comment by yolain** [comment]
It is possible that the node package has a low version of cg-image-picker or other packages that cause compatibility issue.

**Comment by PrometheusDante** [comment]
Can confirm it works now. Thank you very much for the quick fix.

**Comment by nodegraphics** [comment]
Hello,
im still having trouble with cg-image-picker and Easyloader (Full) 
When i try to queue it gets stuck and i have to restart comfyui.
when i get rid of cg-image-picker node, im not having any issues.

Im using XL setup.

<img width="966" alt="Screenshot - 2024-05-17 17 23 46" src="https://github.com/yolain/ComfyUI-Easy-Use/assets/10071439/013126ee-cc1d-4070-b6eb-e0896f4d8620">

any ideas ? if it can be resolved ?

**Comment by yolain** [comment]
> Hello,
> 
> im still having trouble with cg-image-picker and Easyloader (Full) 
> 
> When i try to queue it gets stuck and i have to restart comfyui.
> 
> when i get rid of cg-image-picker node, im not having any issues.
> 
> 
> 
> Im using XL setup.
> 
> 
> 
> <img width="966" alt="Screenshot - 2024-05-17 17 23 46" src="https://github.com/yolain/ComfyUI-Easy-Use/assets/10071439/013126ee-cc1d-4070-b6eb-e0896f4d8620">
> 
> 
> 
> any ideas ? if it can be resolved ?

I'll look at it when I have time. Incidentally, easyuse has a node that converts pipe to basic_pipe,no need to connect all output items to the impact pack.

**Comment by nodegraphics** [comment]
> 
> I'll look at it when I have time. Incidentally, easyuse has a node that converts pipe to basic_pipe,no need to connect all output items to the impact pack.

awesome, I appreciate it ! and thanks for letting me know about the pipe node. i will replace it for sure.

**Comment by yolain** [comment]
> > I'll look at it when I have time. Incidentally, easyuse has a node that converts pipe to basic_pipe,no need to connect all output items to the impact pack.
> 
> awesome, I appreciate it ! and thanks for letting me know about the pipe node. i will replace it for sure.

I didn't have the issues you mentioned, it could be that your version of cg-image-picker is too low or something else is causing it.

**Comment by nodegraphics** [comment]
> 
> I didn't have the issues you mentioned, it could be that your version of cg-image-picker is too low or something else is causing it.

himm, i will check again and see if i can find anytning else. Basically "easyloader (full) gets stuck and doesnt do anythning. i usually have to restart....and i noticed it usually happens with XL models..
